src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -190,10 +190,12 @@ public boolean isDependentArgumentAllowed(final Class<?> dependentClass) {
     // Return the allowable values for readFilterNames/disableReadFilter
     @Override
     public Set<String> getAllowedValuesForDescriptorArgument(final String longArgName) {
-        if (longArgName.equals(StandardArgumentDefinitions.READ_FILTER_LONG_NAME) ||
-                longArgName.equals(StandardArgumentDefinitions.DISABLE_READ_FILTER_LONG_NAME)) {
+        if (longArgName.equals(StandardArgumentDefinitions.READ_FILTER_LONG_NAME)) {
             return readFilters.keySet();
         }
+        if (longArgName.equals(StandardArgumentDefinitions.DISABLE_READ_FILTER_LONG_NAME)) {
+            return toolDefaultReadFilters.keySet();
```

Comment:
I think we should allow the user to disable filters that are already disabled, and enable filters that are already enabled, with just a warning. This allows pipeline authors to guarantee that a certain filter will be on/off, regardless of how the tool defaults change over time. I think the only time we should blow up is if the user both enables and disables the same filter on the command line.

Is this consistent with current behavior @cmnbroad? It's not clear to me that this method is only generating allowed values for display purposes (as opposed to validation purposes) -- can we make this clearer, if it's the case?